benchmark/pkg/runselector/schemes/last_n_runs_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schemes
+
+import (
+	"testing"
+)
+
+var (
+	lastNhoursJob = "ci-kubernetes-e2e-gce-scalability"
+	NumHours      = 3
+)
+
+func TestGetLastNJobRuns(t *testing.T) {
```

Comment:
I'm sorry I wrongly named the variable as `NumHours` as part of copy-pasting. Changed it to (and what I meant was) `numRuns`. All we check is that the function can get the last 3 runs for that test (and has nothing to do with hours as this is the test for --last-n-runs selection scheme). For the --last-n-hours test, we check that we get at least one run in the last 24 hours for the job.